### PR TITLE
Adds information on the groups claim in the access token.

### PR DESCRIPTION
**Pull request by danielzeller-okta**
Primary reviewers: @larsjohansen-okta @mystiberry-okta 

@federations-okta

### CODE REVIEW

**Comment by larsjohansen-okta** [comment]
🚢

**Comment by mystiberry-okta** [comment]
Should I push out right away or wait for .32? Thanks!

File: _source/docs/api/resources/oidc.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ Instead, the ID token contains the `name` and `preferred_username` claims if the
 
 * The full set of claims for the requested scopes is available via the [/oauth2/v1/userinfo](#get-user-information) endpoint. Call this endpoint using the Access Token.
 
-* To protect against arbitrarily large numbers of groups matching the group filter, the group claim has a limit of 100. 
+* To protect against arbitrarily large numbers of groups matching the group filter, the groups claim has a limit of 100.
```
Comment (raphaellondner-okta) [inline _source/docs/api/resources/oidc.md]: It is my understanding that this limitation also applies to the groups claim in the Access Token, so maybe we should add it in  _source/docs/api/resources/oauth2.md as well?

Comment (danielzeller-okta) [inline _source/docs/api/resources/oidc.md]: This is true. I'll add it to the callout as well.

**Comment by danielzeller-okta** [comment]
@mystiberry-okta wait for .32 (cut from today's codefreeze, right?)

**Pull request merged**